app/config.py

```diff
@@ -108,6 +108,8 @@ class Config(object):
 
     # Zendesk
     ZENDESK_API_KEY = os.getenv('ZENDESK_API_KEY')
+    ZEN_DESK_SELL_API_URL = os.getenv('ZEN_DESK_SELL_API_URL')
+    ZEN_DESK_SELL_API_KEY = os.getenv('ZEN_DESK_SELL_API_KEY')
 
     # Freshdesk
     FRESH_DESK_PRODUCT_ID = os.getenv('FRESH_DESK_PRODUCT_ID')
```

Comment:
Can you cast to an int here? I think we're doing it later in the flow

Reply:
Sorry - i'm running into issues here, I'm wondering if this is the wisest recommendations.
Now within celery tasks locally i'm seeing that if FRESH_DESK_PRODUCT_ID is not defined the entire config.py is invalid because it will throw an exception. Maybe best to keep config.py resilient so that constants can be defined as `Optional[str]` - at this point I'm reverting these changes locally